inventories/README.adoc

```diff
@@ -255,3 +256,52 @@ create some file to simulate an API call to provision a host ------------ 0.57s
 +
 TIP: if for some reason, you can't follow the recommendation, you can at least avoid duplicating too much information by indirectly referencing the hosts' variables as in `"{{ hostvars[item.name]['provision_value'] }}"`. Not so bad...
 ====
+
+== Restrict your usage of variable types
+[%collapsible]
+====
+Explanations::
+* Do not use playbook and play variables, as well as include_vars, prefer inventory variables.
```

Comment:
```suggestion
* Avoid playbook and play variables, as well as `include_vars`. 
  Opt for inventory variables instead.
```

Reply:
I won't comment on all your... comments but I'm taking them over in one big commit, and not one by one. I'm resolving them as I include them (without further... comment).